contents/handbook/growth/sales/product-led-sales.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Product-led Sales
+sidebar: Handbook
+showTitle: true
+---
+
+A large proportion of our paying customer base sign up to a paid plan without ever talking to the Sales team.  We don't want to force these customers through a sales process if they don't need it, but we also know that having a human to help them through the process on hand is likely to maximize the chances of retaining them as a paying customer long term.
+
+> We think that product-led sales is going to be extremely important to how PostHog goes, so we will be iterating on this process a lot as we learn. Feel free to edit this page at any time with things that you have found work/don't work. 
+
+## Product-led lead generation
+
+Product-led leads can be generated in different ways:
+
+1. Customers who have an active subscription in Stripe, and are forecasted to pay a first time bill of $500 or more.
+2. Customers who have a high [lead score](/handbook/growth/sales/lead-scoring) (>=20), and have more than 1 user in the organization, but haven't yet subscribed.
```

Comment:
Is having more than one user how Growth define activation as well? If not, should we be consistent here with them?

Reply:
I don't think we need to align with the definition of activation as our goal is for the teams to possibly help high-potential customers to become activated and then subscribe.  By going further downstream we risk ignoring some people with money to spend who need some help.  The reason I specified more that one person in the org is that from prior experience we do have a lot of people who sign up and create duplicate orgs, and also sign up and never get through onboarding.  By narrowing down to having more than one user we end up focusing on people who have the intent to share PostHog with their teammates.  This will create a big top of funnel and we might need to refine but I'd rather a large set of data to learn from for this intitial attempt.

Reply:
Should  we include subscribed (but not yet paid, at least for a few period) accounts here? I think it's pretty common for big teams to start small, one product etc, and then make an expansion decision. 

As we get more strategic about going deep with orgs/cultivating champions, finding expansion. I think it'll be easier for us to prove we did something when an account _doesn't_ have an imminent $500 or $1600 bill coming up. eg they were static for three months < paid threshold, then we came in and found the expansion opp.

Reply:
fyi we have ~100 users that match "high lead score, no stripe id, >1 org members" criteria (planning to turn it on tomorrow after i fix a couple filtering issues)

Reply:
This feels like a good number to split across 4 AEs for now but let's retro this in a few weeks after they've had a chance to explore.  @camerondeleone if you want to proactively go after others without it harming your existing book then go for it but be prepared to share what you learn 👍